task1/Program.cs

```diff
@@ -45,5 +71,792 @@ public static decimal PerformCalculation(int r)
+                    { result = operand1 / operand2; }
+                    break;
+                default:
+                    break;
+            }
+            Console.WriteLine($"{operand1} {sign} {operand2} = {result}");
+        }
+
+        public static void GetNumberArea()
+        {
+            Console.WriteLine("Input number from 0 to 100");
+            int number = Convert.ToInt32(Console.ReadLine());
+            if (number >= 0 && number <= 14)
+            {
+                Console.WriteLine($"{number} is in area [0-14]");
+            }
+            if (number >= 15 && number <= 35)
+            {
+                Console.WriteLine($"{number} is in area [15-35]");
+            }
+            if (number >= 36 && number <= 50)
+            {
+                Console.WriteLine($"{number} is in area [36-50]");
+            }
+            if (number >= 50 && number <= 100)
+            {
+                Console.WriteLine($"{number} is in area [50-100]");
+            }
+        }
+
+        public static void Translator()
+        {
+            Console.WriteLine("Input russian word");
+            string inputString = Console.ReadLine();
+            string translation = "";
+            switch (inputString)
+            {
+                case "дождь":
+                    translation = "rain";
+                    break;
+                case "снег":
+                    translation = "snow";
+                    break;
+                case "ясно":
+                    translation = "sunny";
+                    break;
+                case "туман":
+                    translation = "mist";
+                    break;
+                case "ветер":
+                    translation = "wind";
+                    break;
+                case "штиль":
+                    translation = "still";
+                    break;
+                case "шторм":
+                    translation = "storm";
+                    break;
+                case "торнадо":
+                    translation = "tornado";
+                    break;
+                case "ливень":
+                    translation = "shower";
+                    break;
+                case "облака":
+                    translation = "cloud";
+                    break;
+                default:
+                    Console.WriteLine("This word has no translation in this app");
+                    return;
+            }
+            Console.WriteLine($"{inputString} in English is {translation}");
+        }
+
+        public static void BonusCalculation()
+        {
+            Console.WriteLine("Input sallary");
+            double salary = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("Input stage");
+            int stage = Convert.ToInt32(Console.ReadLine());
+            double bonus = 0;
+
+            if (stage < 5)
+            {
+                bonus = 0.1;
+            }
+            if (stage >= 5 && stage < 10)
```

Comment:
The same as above - the stage could belong to the only range, so no need to check further conditions if on previous was succeeded.
Please use else if statements.